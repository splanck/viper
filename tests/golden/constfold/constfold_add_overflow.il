il 0.1.2
extern @rt_print_i64(i64) -> void
func @main() -> i64 {
entry:
  %t0 = iadd.ovf 9223372036854775807, 1
  call @rt_print_i64(%t0)
  ret 0
}
