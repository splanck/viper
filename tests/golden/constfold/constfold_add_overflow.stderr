runtime trap: Overflow @ main: entry[#0]: integer overflow in iadd.ovf
